core/domain/feedback_services_test.py

```diff
@@ -87,11 +86,13 @@ class SuggestionQueriesUnitTests(test_utils.GenericTestBase):
     def _generate_thread_id(self, unused_exp_id):
         return self.THREAD_ID1
 
-    def _get_threads(self, unused_exploration_id):
-        return self.threads
+    def _put_threads(self):
```

Comment:
The reason was one test using "_put_threads_with_suggestions", but I don't see any difference using _put_threads instead.